.github/workflows/ci.yml.jinja

```diff
@@ -35,13 +35,13 @@ jobs:
         python:
           - version: '{{min_python}}'
             tox-env: 'py{{min_python|replace(".", "")}}'
-{% raw %}
+{% raw -%}
     uses: ./.github/workflows/test.yml
```

Comment:
This messes up the indentation.
I don't know the best way to avoid it but something like this should work:
```.yml
           tox-env: 'py{{min_python|replace(".", "")}}'{% raw %}
    uses: ./.github/workflows/test.yml
```

But in general, are the extra empty lines really an issue? I wouldn't expect us to look much at the rendered files anyway.

Reply:
Probably not an issue no, I was just confused in some cases, because there were empty lines before the params of an action.